app/controllers/solidus_paypal_commerce_platform/orders_controller.rb

```diff
@@ -5,6 +5,24 @@ class OrdersController < ::Spree::Api::BaseController
     before_action :load_order, except: :create
     before_action :load_payment_method, only: :show
     skip_before_action :authenticate_user
+    include ::Spree::Core::ControllerHelpers::Auth
+
+    def create
+      authorize! :create, ::Spree::Order
+
+      @order = ::Spree::Order.create!(
+        user: try_spree_current_user,
```

Comment:
shouldn't this be `current_api_user` and not need the inclusion of `::Spree::Core::ControllerHelpers::Auth`?

Reply:
I'm looking into this now, but `current_api_user` returns nil even when I'm logged in - maybe because we're skipping `authenticate_user` in favor of authorizing based on the order token (since frontend users may not have an API key). I'm not entirely sure though.

Reply:
Yeah, looks like it's based on the `api_key`, which users may not have even when logged in.
https://github.com/solidusio/solidus/blob/8b37c63c2914e9b00409d272b4c68eab54172a84/api/app/controllers/spree/api/base_controller.rb#L46-L48